tests/unit/test_reconciler.py

```diff
@@ -38,52 +38,47 @@ def full_config(minimal_config):
     return minimal_config
 
 
-args = 'config_type,timeout,provide_loop'
+@pytest.fixture
+def dns_client(mocker, monkeypatch):
+    # creds = mocker.Mock(auth.GoogleAuthClient, autospec=True)
+    mock = mocker.Mock(cloud_dns.AIOGoogleDNSClient, autospec=True)
+    mock._session = mocker.Mock()
+    mock._session.close.return_value = True
+    monkeypatch.setattr('gordon_janitor_gcp.cloud_dns.AIOGoogleDNSClient', mock)
+    return mock
+
+
+args = ',timeout,'
```

Comment:
How does this work with the `params` for parametrize? There's only one keyword ('timeout'), but 2 params for each run (`[False, False]`, `[80, True]`).